### PR TITLE
Update 2015-06-25-anova-test.md

### PR DESCRIPTION
**Pull request by corinne-riddell**
@mkcor ANOVA page updated. I updated the formatting of the plot that I embedded in the file but it doesn't seem to reflect the updates.. any ideas?

### CODE REVIEW

**Comment by corinne-riddell** [comment]
@mkcor ... the formatting of the plot is updated now so nevermind that!

File: _posts/2015-06-25-anova-test.md [hunk 1]
```diff
@@ -32,14 +32,16 @@ Both plots show the raw data as well as the average donation amounts (indicated
...
 
-Sometimes, you might be convinced by graphing the data alone that there is a difference among the group means, such as with the data shown for Scenario 1. However, you can imagine an additional scenario in which the visualization isn’t so clear; there might be a difference between the means, but based on visualization alone it is difficult to gauge whether we think there is a difference or not. Thus, tests such as ANOVA provide objective tools to quantify the size of the difference.
+Sometimes, you might be convinced by graphing the data alone that there is a difference among the group means, such as with the data shown for Scenario 1. However, you can imagine an additional scenario in which the visualization isn’t so clear; there might be a difference between the means, but based on visualization alone it is difficult to gauge whether we think there is a difference or not. ANOVA is an objective tool used to say whether a difference exists or not.
```
Comment (mkcor) [inline _posts/2015-06-25-anova-test.md]: Trailing whitespace.

File: _posts/2015-06-25-anova-test.md [hunk 2]
```diff
@@ -32,14 +32,16 @@ Both plots show the raw data as well as the average donation amounts (indicated
...
+The one-way ANOVA test uses information about how far each group average is away from the overall average to quantify differences across the groups. The test statistic is a ratio of two numbers, where the numerator quantifies the amount of variability between group averages and the denominator quantifies the amount of varability within each group. A higher test statistic value implies that the numerator is large and that there is more variabiity between group averages which implies a difference in the average value across groups. Thus, the higher the value of the test statistic, the more evidence there is against the null hypothesis.
```
Comment (mkcor) [inline _posts/2015-06-25-anova-test.md]: Typo (varability --> variability).

File: _posts/2015-06-25-anova-test.md [hunk 3]
```diff
@@ -58,9 +60,9 @@ In the numerator of the ANOVA test statistic, we have the estimate for the varia
...
 
-We haven’t said much about the denominators G-1 and \sum\limits_{g=1}^G(n_g-1) of the test statistic. Without going into too much detail, these denominators are used to provide averages. For example G-1 helps us estimate the average squared distance between the group means and the overall mean, while the second one, \sum\limits_{g=1}^G(n_g-1) helps us estimate the average squared distance within a group. These quantities are known as the degrees of freedom of the numerator and the degrees of freedom of the denominator, respectively.
+We haven’t said much about the quantities $G-1$ and $\sum\limits_{g=1}^G(n_g-1)$ found in the ANOVA test statistic. Without going into too much detail, these terms are used to provide averages for the numerator and the denominator. For example $G-1$ helps us estimate the average squared distance between the group means and the overall mean, while the second quantity, $\sum\limits_{g=1}^G(n_g-1)$ helps us estimate the average squared distance within a group. These quantities are known as the degrees of freedom of the numerator and the degrees of freedom of the denominator, respectively.
```
Comment (mkcor) [inline _posts/2015-06-25-anova-test.md]: Maybe this is on purpose because the mathematical expressions are in line but, precisely, I would drop `\limits`.

Comment (mkcor) [inline _posts/2015-06-25-anova-test.md]: Drop 'the' before 'quantities'?

File: _posts/2015-06-25-anova-test.md [hunk 4]
```diff
@@ -12,7 +12,7 @@ actioncall: Make a Graph
 
 # The One-way ANOVA Test
 
-## When do you use one-way ANOVA?
+### When do you use one-way ANOVA?
 ANOVA stands for the “analysis of variance”. One-way ANOVA is a statistical test used to determine if there are differences between three or more groups on **one** continuous outcome of interest. For example:
```
Comment (mkcor) [inline _posts/2015-06-25-anova-test.md]: Drop 'the'.

File: _posts/2015-06-25-anova-test.md [hunk 5]
```diff
@@ -58,9 +60,9 @@ In the numerator of the ANOVA test statistic, we have the estimate for the varia
...
+We haven’t said much about the quantities $G-1$ and $\sum\limits_{g=1}^G(n_g-1)$ found in the ANOVA test statistic. Without going into too much detail, these terms are used to provide averages for the numerator and the denominator. For example $G-1$ helps us estimate the average squared distance between the group means and the overall mean, while the second quantity, $\sum\limits_{g=1}^G(n_g-1)$ helps us estimate the average squared distance within a group. These quantities are known as the degrees of freedom of the numerator and the degrees of freedom of the denominator, respectively.
 
-If the null hypothesis is false, the F statistic will be a large number as the variance between groups (the numerator) overcomes the variance within the groups (the denominator). Let’s go through an example with the data from Scenario 1 to see how the calculation works:
+If the null hypothesis is false, the F-statistic will be a large number as the variance between groups (the numerator) overcomes the variance within the groups (the denominator). Let’s go through an example with the data from Scenario 1 to see how the calculation works:
```
Comment (mkcor) [inline _posts/2015-06-25-anova-test.md]: Drop 'the' before 'groups'?

File: _posts/2015-06-25-anova-test.md [hunk 6]
```diff
@@ -78,14 +80,14 @@ If the null hypothesis is false, the F statistic will be a large number as the v
 |**Group Average**| **100.20** | **123.50**| **129.60**|
 |**Overall Average**| **117.77**| | |
 
-For the numerator of the test statistic we take the squared difference between each group average and the overall average and multiply by the number of people in each group. We then divide by the degrees of freedom for the numerator, which is equal to G-1:
+For the numerator of the test statistic we take the squared difference between each group average and the overall average and multiply by the number of people in each group. We then divide by the degrees of freedom for the numerator, which is equal to $G-1$:
```
Comment (mkcor) [inline _posts/2015-06-25-anova-test.md]: '... by the number of degrees of freedom of the numerator, which is equal...'

File: _posts/2015-06-25-anova-test.md (hunk 4 above)
Comment (mkcor) [inline _posts/2015-06-25-anova-test.md]: 'continuous outcome of interest'? There either too little or too much. It's either too mathematical or not enough.
- 'outcome of interest'?
- 'continuous variable describing our groups'?
- ...

File: _posts/2015-06-25-anova-test.md [hunk 7]
```diff
@@ -103,7 +105,7 @@ For the denominator of the test statistic we take the squared difference between
...
+F = 2407/22.02 = 109.3. This number is very large because the numerator was very big compared with the denominator. The last step is to determine if this difference is large enough to provide evidence against the null hypothesis. To do that, we compare the F-statistic to a statistical distribution known as the F distribution. The F distribution also requires that you provide the corresponding degrees of freedom for the numerator and denominator. Remember that these were equal to 2 and 27, respectively. We can ask Plotly what is the chance of observing an F-statistic larger than or equal to 109.3 in the situation where there is no difference between the campaigns in the average amount donated. This probability (known as the p-value) is very tiny for this example - smaller that 0.01% - meaning that it is extremely unlikely you’d observe an F-statistic this small if there was truly no difference between the campaigns. Here, we would conclude that there is strong evidence against the null hypothesis of no difference between the campaigns.
```
Comment (mkcor) [inline _posts/2015-06-25-anova-test.md]: Maths: $\text{F} = 2407/22.02 = 109.3$

File: _posts/2015-06-25-anova-test.md [hunk 8]
```diff
@@ -103,7 +105,7 @@ For the denominator of the test statistic we take the squared difference between
...
 
 ### Be careful...
 In order to use ANOVA, our data should follow certain assumptions or else our inference based on the ANOVA test is more likely to be incorrect. The most important assumption is that the observations are independent of one another. It should be the case that each sample is an independent random sample. In our example, this assumption would be satisfied if the sponsors were randomly sampled from a database of all potential sponsors. Thus, to understand whether this assumption is met you need to know how the data was collected. The second assumption of ANOVA is that the variance of the observations within each group is the same across the groups. The third assumption is that the data follow a normal distribution (a bell curve) within each group. Fortunately, the ANOVA test tends to perform well even if this last assumption is violated, especially in the case where the sample size per group is large.
```
Comment (mkcor) [inline _posts/2015-06-25-anova-test.md]: Drop the 'or else ...' part.

Comment (mkcor) [inline _posts/2015-06-25-anova-test.md]: Drop 'the' before 'observations'.

File: _posts/2015-06-25-anova-test.md (hunk 7 above)
Comment (mkcor) [inline _posts/2015-06-25-anova-test.md]: 'what is the chance' --> 'what the chance is'

**Pull request merged**